crates/krator/src/runtime.rs

```diff
@@ -102,6 +72,26 @@ impl<O: Operator> OperatorRuntime<O> {
             operator: Arc::new(operator),
             list_params,
             signal: None,
+            store: Store::new(),
+        }
+    }
+
+    pub(crate) fn new_with_store(
```

Comment:
Instead of a new constructor, would it make sense to follow a builder pattern here? As in should we provide a new OperatorBuilder type that looks like this:

```
let builder = Operator::builder().add_store(Store::new()).build()?;
```

Kinda like https://docs.rs/reqwest/0.11.2/reqwest/struct.ClientBuilder.html

That way we won't have 1,000,001 different `new_with_*` functions as part of the API.

I see we're already following this pattern with the ControllerBuilder API.

Reply:
I would normally agree, but I think the public API for this will disappear soon, and the `new` method will be changed require the `Store` argument as well.